src/strings/PowerFxResources.en-US.resx

```diff
@@ -4853,4 +4889,7 @@
       Some examples for valid named type declarations - "Point := Type({x: Number, y: Number});" , "T1 := Type(Number);" , "T2 := Type([Boolean]);".
     </comment>
   </data>
+  <data name="ErrJoinCantUnion" xml:space="preserve">
+    <value>Data sources can't union. Possibly due to duplicated columns.</value>
```

Comment:
This error message is too "dry". We should try to give some possible solution to the customer.

Reply:
I changed the error message to include the duplicated columns.